content/site/callForMacroscopes.md

```diff
@@ -66,61 +73,70 @@ tabs:
 
 
       * Link to online site that features the macroscope tool or link to executable code. This must be a fully self-contained version of the macroscope that can operate without any outside links and without opening new windows.
 
       * Biographies for all authors (100 words each)
 
-      * High resolution portraits of all authors that are no smaller than 360 x 450 pixels, or 1.2" x 1.5" at 300 dpi.
+      * High resolution author portraits that are no smaller than 360 x 450 pixels, or 1.2” x 1.5” at 300 dpi.
 
       * Signed copyright and reproduction agreement
 
 
-      Authors are welcome to use the expertise and resources of the exhibit curators and designers. The macroscopes are expected to be ready for display by August 31, 2021.
 
 
-      #### Exhibit Advisory Board
+      Authors are welcome to use the expertise and resources of the exhibit curators and designers. The macroscopes are expected to be ready for display by August 31, 2022.
+
+
+      ## Exhibit Advisory Board
 
 
       * Gary Berg-Cross, Cognitive psychologist
 
+      * Kevin Boyack, President, SciTech Strategies Inc.
+
       * Donna Cox, Advanced Visualization Laboratory, University of Illinois at Urbana-Champaign
 
       * Bonnie DeVarco, Media X, Stanford University
 
-      * Ingo Günther, Karlsruhe Univerity of Art and Design
+      * Ingo Günther, Karlsruhe University of Art and Design
 
       * Francis Harvey, Cartography and Visual Communication, Leipzig University
 
-      * Peter A. Hook, LawArXiv, Cornell Law Library
+      * Peter A. Hook, Associate Law Librarian, University of Notre Dame
 
-      * Lev Manovich, Computer Science, The Graduate Center, City University of New York
+      * Vincent Lariviére, Professor of Information Science, Universitè de Montrèal
 
-      * Elijah Meeks, Netflix
+      * Lev Manovich, Computer Science, The Graduate 
+
+      * Elijah Meeks, Chief Visualization Officer & Co-founder, Noteable
 
       * André Skupin, Geography, San Diego State University
 
-      * Olga Subiros, Big Bang Data, Olga Subiros Studio
+      * Olga Subirós, Big Bang Data, Olga Subiros Studio
+
+      * Stephen Uzzo, Chief Scientist, New York Hall of Science
 
-      * Stephen Uzzo, New York Hall of Science
+      * Caroline S. Wagner, Wolf Chair in International Affairs, The Ohio State University
 
       * Benjamin Wiederkehr, Interactive Things
 
 
-      Please feel free to send any questions you might have regarding the judging process to Katy Börner (ude.anaidni@ytak) and use the subject heading “Macroscope Inquiry.”
+
+
+      Please feel free to send any questions you might have regarding the judging process to Todd Theriault ([ttheriau@indiana.edu](mailto:ttheriau@indiana.edu)) and use the subject heading “Macroscope Inquiry.”
 importantDates:
-  - date: 2021-03-15T19:22:23.487Z
+  - date: 15 02 22
```

Comment:
Match formatting of the previous date entries.

Reply:
will add the dates manually in a second, and correct the iteration number

Reply:
it is not allowing me to change the date format manually. I had to choose from the incorporated calendar.